app/content/[...slug]/page.tsx

```
@@ -0,0 +1,411 @@
+                    category: false,
+                    tags: false
+                  }}
+                />
+              </div>
+            </div>
+          </header>
+
+          {/* Article content */}
+          <div 
+            className="prose prose-lg max-w-none dark:prose-invert"
+            dangerouslySetInnerHTML={{ __html: content.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)